.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,43 @@
+Fixes: <https://github.com/mozilla/addons/ISSUENUM>
+
+<!--
 Thanks for opening a Pull Request (PR), here's a few guidelines as to what we need in your PR before we review it.
+-->
+
+### Description
+
+<!--
+Your PR will be squashed when merged so the 1st commit must contain a descriptive and concise summary of the change.
+Additional details should be added in the description. If your change is simple enough to summarize in the commit, or
+if it is not relevant for your PR, remove this section.
+-->
+
+### Context
+
+<!--
+Often a pull request contains changes that are not fully self explanatory. Maybe this PR is a part of a series,
+or maybe it is a partial change now with a more ambitious plan for the future. Add this additional context here.
+If it is not relevant for your PR, remove this section.
+-->
+
+### Testing
+
+<!--
+Your Change must be related to an existing, open issue. This issue should contain testing instructions.
```

Comment:
```suggestion
Your change must be related to an existing, open issue. This issue should contain testing instructions.
```
I guess?  I wasn't sure if "Change" was supposed to be a proper noun in this template, but it's not capitalized in other places.